.github/workflows/e2e.yml

```diff
@@ -45,45 +45,48 @@ jobs:
       SYSTEM_TEST_JS_STELLAR_SDK_GH_REF: 
 
       # the version of rs-stellar-xdr to use for quickstart
-      SYSTEM_TEST_RS_XDR_GIT_REF: v20.0.2
+      SYSTEM_TEST_RS_XDR_GIT_REF: v20.1.0
 
       # system test will build quickstart image internally to use for running the service stack
       # configured in standalone network mode(core, rpc)
-      SYSTEM_TEST_QUICKSTART_GIT_REF: https://github.com/stellar/quickstart.git#412bb828ddb4a93745227ab5ad97c623d43f3a5f
+      SYSTEM_TEST_QUICKSTART_GIT_REF: https://github.com/stellar/quickstart.git#eeb44ee99c564275654800edea2e426d73e966b8
 
       # triggers system test to log out details from quickstart's logs and test steps
       SYSTEM_TEST_VERBOSE_OUTPUT: "true"
 
       # the soroban test cases will compile various contracts from the examples repo
-      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH: "v20.0.0"
+      SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_HASH: "v20.2.0"
       SYSTEM_TEST_SOROBAN_EXAMPLES_GIT_REPO: "https://github.com/stellar/soroban-examples.git"
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         name: checkout system-test
         with:
           repository: stellar/system-test
           ref: ${{ env.SYSTEM_TEST_GIT_REF }}
           path: system-test
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
         name: checkout soroban-tools
         with:
           repository: stellar/soroban-tools
           path: soroban-tools
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
I think can remove this local checkout of soroban-rpc, SYSTEM_TEST_SOROBAN_RPC_REF provides where that source is coming from.